packages/arb-token-bridge-ui/src/components/TransferPanel/LowBalanceDialogContent.ts

```diff
@@ -3,38 +3,32 @@ export const LowBalanceDialogContent = {
     {
       href: 'https://www.binance.com',
       title: 'Binance',
-      imageSrc:
-        'https://portal.arbitrum.one/wp-content/uploads/2021/11/i0KjbCry_400x400.jpg'
+      imageSrc: '/images/projects/binance-logo.webp'
     },
     {
       href: 'https://www.bitget.com',
       title: 'Bitget',
-      imageSrc:
-        'https://portal.arbitrum.one/wp-content/uploads/2022/02/bitget.jpeg'
+      imageSrc: '/images/projects/bitget-logo.webp'
     },
     {
       href: 'https://www.bybit.com',
       title: 'Bybit',
-      imageSrc:
-        'https://portal.arbitrum.one/wp-content/uploads/2022/01/20220126-215931.png'
+      imageSrc: '/images/projects/bybit-logo.webp'
     },
     {
       href: 'https://crypto.com',
       title: 'Crypto.com',
-      imageSrc:
-        'https://portal.arbitrum.one/wp-content/uploads/2021/11/404-4048020_crypto-com-logo-svg-hd-png-download.png'
+      imageSrc: '/images/projects/crypto-com-logo.webp'
     },
     {
       href: 'https://www.huobi.com',
```

Comment:
Huobi is no longer on our Portal so we should remove it here, too. Please double check the other projects, too

Reply:
Updated, all other are still present in the portal. I also removed the analytic, do we want to delete it from fathom or do we want to keep it for now?